AppCreationScripts/apps.json

```diff
@@ -9,19 +9,13 @@
       "x-ms-id": "active-directory-android-java",
       "x-ms-name": "android-java",
       "x-ms-version": "1.0",
-      "replyUrlsWithType": [
-        {
-          "url": "msauth://{Enter_the_Package_Name}/{Enter_the_Signature_Hash}",
-          "type": "InstalledClient"
-        }
-      ],
       "requiredResourceAccess": [
         {
           "x-ms-resourceAppName": "Microsoft Graph",
           "resourceAppId": "00000003-0000-0000-c000-000000000000",
           "resourceAccess": [
             {
-              "id": "Enter_the_Application_Id_here",
+              "id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
```

Comment:
Where are you getting this id from?

Reply:
In the apps.json file, the resource App Id was initially set to "e1fe6dd8-ba31-4d61-89e7-88639da4683d" which is the correct resourceAppId for User.Read. The resourceAppID is set to "Enter_the_application_Here" here. The application AppId shouldn't be the resouce App ID in requiredResourceAccess. If the required resource is User.Read of MSGraph then the resource App Id should be set to "e1fe6dd8-ba31-4d61-89e7-88639da4683d".

Reply:
fyi: https://marketplace.visualstudio.com/items?itemName=stephane-eyskens.aadv1appprovisioning

e1fe6dd8-ba31-4d61-89e7-88639da4683d is associated to MSGraph's User.Read